platform/golden/cfg/preseed-debian-11.cfg

```diff
@@ -1,22 +1,18 @@
+#_preseed_V1
+
 ####################
 ## General defaults
 
-d-i debian-installer/locale string en_US.UTF-8
-d-i kbd-chooser/method select us
-d-i localechooser/supported-locales multiselect en_US.UTF-8
-d-i locale select en_US.UTF-8
+d-i debian-installer/locale string en_US
 d-i keyboard-configuration/xkb-keymap select us
-d-i console-keymaps-at/keymap select us
-d-i console-setup/ask_detect boolean false
-d-i debconf/frontend select Noninteractive
 
 ####################
 ## Network defaults
 
 d-i netcfg/choose_interface select auto
 d-i netcfg/get_hostname string golden
-d-i netcfg/get_domain string local
-d-i anna/choose_modules string network-console
+d-i netcfg/get_domain string artificialwisdom.cloud
+d-i netcfg/wireless_wep string
```

Comment:
do we need WEP config? I'd be surprised if we have a WIFI adapter available?

Reply:
web config is not needed. maybe ages ago, and nobody has updated the docs. Nobody including me :)